tf2/include/tf2/buffer_core.h

```diff
@@ -444,6 +444,6 @@ class BufferCore
 
 
 
-};
+}
```

Comment:
I confess I'm not too interested in fixing this for ROS1. My team at ASI isn't using ROS1, and I'm not too familiar with it.